ichub-backend/services/sharing_service.py

```diff
@@ -27,8 +27,8 @@
 from uuid import UUID
 from managers.submodels.submodel_document_generator import SubmodelDocumentGenerator, SEM_ID_PART_TYPE_INFORMATION_V1
 from managers.metadata_database.manager import RepositoryManagerFactory, RepositoryManager
-from models.services.twin_management import CatalogPartTwinCreate, CatalogPartTwinShare, TwinAspectCreate, CatalogPartTwinDetailsRead, TwinAspectRead
-from models.metadata_database.models import BusinessPartner, DataExchangeAgreement, EnablementServiceStack, CatalogPart, Twin, PartnerCatalogPart
+from models.services.twin_management import CatalogPartTwinCreate
```

Comment:
```suggestion
from models.services.twin_management import CatalogPartTwinCreate, TwinAspectCreate
```
It is used on line 80.